internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,64 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	data := strings.Split(datastring, ",")
+	if len(data) != 2 {
+		return fmt.Errorf("invalid input format: expected 2 values")
+	}
+
+	steps, err := strconv.Atoi(data[0])
+	if err != nil {
+		return fmt.Errorf("invalid number of steps: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("invalid number of steps: %w", err)
```